web/src/main/webapp/dataViews/heavyHittersView.ts

```diff
@@ -234,6 +245,24 @@ export class HeavyHittersView extends RemoteTableObjectView {
             str = SpecialChars.approx + str;
         return str;
     }
+
+    private changeThreshold(): void {
+        let min: number = 0.1;
```

Comment:
I suspect this value appears somewhere else as well; we should have only one occurrence of this constant.